lib/network/modules/components/nodes/shapes/CircularImage.js

```diff
@@ -5,17 +5,16 @@ import CircleImageBase from '../util/CircleImageBase'
 /**
  * A CircularImage Node/Cluster shape.
  *
- * @class CircularImage
+ * @param {Object} options
```

Comment:
There are constructor parameters. Is there a valid (perhaps documented) reason to place them here?

This applies to all further occurrences.